### ISSUE TITLE
List document for rejected brands

### ISSUE THREAD

**Issue by jorgeamadosoria** [simple-icons/simple-icons#6394]
### Kind of issue 
  - [X] Improvement
  - [ ] Mistake
  - [ ] Other, namely:

### Description
As discussed in https://github.com/simple-icons/simple-icons/issues/6350#issuecomment-904199882, we need a text document, json file, or other way to keep a clear and easily accesible track of the brands that have already being rejected, as well as the issue where that brand's rejection was handled.

Some brands may have been rejected or removed due to changing circumstances (Alexa rankings, for example) but other may have been banned for license reasons (Marvel), guidelines (Bundeswehr) or technical limitations of the library (monday.com).
These causes would be useful to list so the would-be contributor looking to propose them again would know why they were rejected, thus reassessing the approach to request further icons (in my experience, people tend to request items in batches) and also giving them the chance to appeal the rejection if the icon's circumstances have changed (for example, monday.com could allow a monochrome logo version at some point in the future).

The list can take the form of a Markdown table in an REJECTED.md file (or some other, less negative name), that includes the brand name, the issue number in which it was first mentioned, and a one sentence reason as to why it wasn't added (so not to force the user to go over all the discussion if they don't have to).
It could also take the form of a JSON file that can be interpreted by the website to render the list in a PUG template. 

Having this list handy can help reducing issue duplication, reduces the time need by maintainers to answer and justify the rejection of a brand more than once.

It also could potentially led to easy automation of issue evaluation by a bot. A list is easily scannable by a bot that could be added as a check in issues, to see if the issue is duplicated, similar to the proposed Alexa Ranking bot for the same purpose. This is a different issue, however, although it requires this list as a pre-requisite.

In short, I propose a document that contains all rejected brands so far, the issue in which discussion happened, and a one sentence summary as to why it happened.

I'm willing to add this file, as well as sift through issues to initialize it with the brands that have already been rejected.

**Comment by jorgeamadosoria** [comment]
> If we want to keep it in a file, we'll need to:
> 
> 1. Explain why in the issue
> 2. Close the issue
> 3. Open a file
> 4. Check if it was already added
> 5. Add or update the entry
> 6. Add the explanation (that we already wrote in the issue)
> 7. Create a PR
> 8. Wait for a review
> 9. Merge it in the repo

steps 6 to 9 are highly optional. Why apply the full flow for something that is entirely bureaucratic?
step 5 is also conditional to the existence or not of a brand rejection.

> Looks to be a lot of steps to duplicate the reason for a rejection. I do not see a good enough benefits for this process.
Also, this allows for automation: with this list, a bot can detect duplicates (avoiding more #6421 requests, for example) or automatically reject brands that have already been rejected.
Even without automation, a list of rejected brands is easy to Ctrl + F by would be contributors before they even try to request something that will be rejected. Nothing stops that right now.
 
> Also rejection often need to be reevaluated, so it is ok for someone to ask again for a previously rejected icon. The Alexa Rank (or Github stars) might have improved since the first request.
Rejections are not set in stone. A reevaluated rejection is one that can be removed from the file. This is fine, and that's the reason why a rejection summary is important.
For example, for Yandex the rejection would be something like "Refused permission by owner".
For Alexa / Github, it would be "Alexa ranking / Github star threshold not met". 
This is good enough to manage this list.

> I'm also not sure it is easier to search a brand name in a big file than to search Github issues. 
It definitely is. Specially if users are directed to do so in the guidelines. 
We don't have a "rejected / removed / deleted" label. The closest thing to that is "won't add" which is not exactly the same. So looking for, say, "Disney" yields a huge list of issues, some open, some labeled as "won't add", but some not labeled in any consistent way. This tells me nothing about Disney without reading at least a couple tickets (since some tickets just refer to previous ones for the rationale). This is messy.
Other times, you get situations like the one with Yandex, where looking for issues only yields no useful information whatsoever about the icon being rejected. The only way to get that info is to look also in PRs, and that yields many results, only one of which (labeled "breaking change", not removal or deletion or rejection) has the info the user would need.

A rejection.md file is just a Ctrl + F distance away of giving a contributor the info they need. There is no contest.

> I'm not sure people will really have a look at it and will probably open a new issue anyway. 
They will if they are directed to. People take the time to look for pages, alexa rankings and all the other info, why not this? And even if they don't, we can use it as internal resource to speed up the rejection of some requests. Not to mention, again, the potential for automated rejection by bot. 

> If maintainer need to close it again, it is easier to link to another issue comments where the rejection was explained than linking to a specific line in a file.
It's the same thing to link to a root issue than to link to a line in a file that in turn links to the root issue.

--------------------------------------------

Alternatively, a label of "rejected / deleted / not added" can also work. It would be a shorthand for searches. It's not as clear, but it's clear enough. "Won't add" is close, but doesn't cover the cases of icons added and later removed. A new label for that would be an acceptable alternative.

**Comment by ericcornelissen** [comment]
> A rejection.md file is just a Ctrl + F distance away of giving a contributor the info they need. There is no contest.

I have to agree that such a file, if implemented properly, can be faster than searching in issues. Additionally, if it is linked it in [the issue templates/forms](https://github.com/simple-icons/simple-icons/tree/develop/.github/ISSUE_TEMPLATE) then anyone we would search through the issues should find this file as well. (that's not to say that I'm no in favor of adding such a file :upside_down_face:)


> Alternatively, a label of "rejected / deleted / not added" can also work. It would be a shorthand for searches. It's not as clear, but it's clear enough. "Won't add" is close, but doesn't cover the cases of icons added and later removed. A new label for that would be an acceptable alternative.

I guess adding a "deleted" tag for the cases not covered by "Won't add" is a good start. Then, the GitHub URL for issues & PRs with either of those to tags would be roughly equivalent to the file proposed in this issue. The only major difference I see is that it _can_ be hard to find the reason for a rejection in a GitHub issue.

**Comment by jorgeamadosoria** [comment]
> > A rejection.md file is just a Ctrl + F distance away of giving a contributor the info they need. There is no contest.
> 
> I have to agree that such a file, if implemented properly, can be faster than searching in issues. Additionally, if it is linked it in [the issue templates/forms](https://github.com/simple-icons/simple-icons/tree/develop/.github/ISSUE_TEMPLATE) then anyone we would search through the issues should find this file as well. (that's not to say that I'm no in favor of adding such a file 🙃)
> 
> > Alternatively, a label of "rejected / deleted / not added" can also work. It would be a shorthand for searches. It's not as clear, but it's clear enough. "Won't add" is close, but doesn't cover the cases of icons added and later removed. A new label for that would be an acceptable alternative.
> 
> I guess adding a "deleted" tag for the cases not covered by "Won't add" is a good start. Then, the GitHub URL for issues & PRs with either of those to tags would be roughly equivalent to the file proposed in this issue. The only major difference I see is that it _can_ be hard to find the reason for a rejection in a GitHub issue.
> 
> I would be very much in favor of this suggestion 😃

IU agree that we lose some info adding the label.
But, in the spirit of consensus, I think it's good enough. With that label one could, potentially, make a search query that shows all icons considered and not included (won't add + deleted) and the same query can be used for automation.
It's not as good as having the file, but it does take less effort. 
If keeping the file is a hard-no for other maintainers, please consider this compromise. I volunteer to go through the issues and retroactively label them appropriately.

My end goal is a bot that can automatically check for icons already considered and close requests automatically if found.

**Comment by jorgeamadosoria** [comment]
given the lack of activity in discussion here, I'm going to go ahead and close this issue and open a different one for adding a "deleted" label.
